suite-native/state/src/store.ts

```diff
@@ -17,18 +16,17 @@ if (__DEV__) {
     middlewares.push(reduxFlipperDebugger);
 }
 
-export const store: Store = configureStore({
-    reducer: rootReducers,
-    middleware: getDefaultMiddleware =>
-        getDefaultMiddleware({
-            thunk: {
-                extraArgument: extraDependencies,
-            },
-            serializableCheck: false,
-            immutableCheck: false,
-        }).concat(middlewares),
-});
+export const initStore = async () =>
+    configureStore({
+        reducer: await prepareRootReducers(),
+        middleware: getDefaultMiddleware =>
+            getDefaultMiddleware({
+                thunk: {
+                    extraArgument: extraDependencies,
+                },
+                serializableCheck: false,
+                immutableCheck: false,
+            }).concat(middlewares),
+    });
 
-export type RootState = ReturnType<typeof store.getState>;
-
-export const storePersistor = persistStore(store);
+export type RootState = ReturnType<Store['getState']>;
```

Comment:
What reason for this change? `Store` is just generic type for any store, it doesn't hold any real information about real shape of state object on our specific case.

Reply:
fixed in de5504964